app/models/ext_management_system.rb

```diff
@@ -167,6 +167,7 @@ def self.create_from_params(params, endpoints, authentications)
   supports_not :add_volume_mapping
   supports_not :admin_ui
   supports_not :assume_role
+  supports_not :auth_key_pair_create
```

Comment:
can we move this to auth_key_pair? I really like sticking with crud/rest verbs where possible

aah, don't think we'd even need these since I'm pretty sure this defaults to false.

```ruby
class AuthKeyPair
  supports_not :create
  supports_not :delete
end
```

Reply:
Discussed in a DM but we need to be able to filter EMSs which are able to create auth_key_pairs so we can't just go off of the AuthKeyPair subclass supporting create or not.